gr-blocks/python/blocks/parse_file_metadata.py

```diff
@@ -75,12 +75,14 @@ def parse_header(p, VERBOSE=False):
         r = pmt.dict_ref(p, pmt.string_to_symbol("rx_time"), dump)
         secs = pmt.tuple_ref(r, 0)
         fracs = pmt.tuple_ref(r, 1)
-        secs = float(pmt.to_uint64(secs))
+        secs = pmt.to_uint64(secs)
         fracs = pmt.to_double(fracs)
-        t = secs + fracs
+        t = float(secs) + fracs
+        info["rx_secs"] = secs
+        info["rx_time_fracs"] = fracs
         info["rx_time"] = t
         if(VERBOSE):
-            print("Seconds: {0:.6f}".format(t))
+            print("Seconds: {0}.{1}".format(secs, str(fracs)[2:]))
```

Comment:
This will no longer print a fixed number of decimal places. Is that intended?

I don't have a strong preference.

Reply:
Yes. This is deliberate. I preferred to print out all the resolution that is stored in the double rather than printing a fixed number of decimal places, since different users might have different needs (though I guess that at least 10 decimal places would be a sensible choice).

However, I don't like the way I did this in Python. Maybe there's a better way.

Reply:
> Maybe there's a better way.

My suggestion would be to use the `decimal` module:

```
>>> import decimal
>>> decimal.getcontext().prec = 20
>>> secs = 1602513164
>>> fracs = 0.15
>>> print(decimal.Decimal(secs) + decimal.Decimal(fracs))
1602513164.1500000000
```
If you want more than 10 decimal places, you can increase `prec`.

Reply:
Thanks! That seems a good suggestion. I've just pushed a commit with my take at using `decimal` for this. In fact I want more than 20 decimal places resolution, since I want to have 1e-16 resolution (which is basically what we can expect from `fracs`) no matter what the value of `secs` is. Apparently this works out to 36 decimal places.